### PR TITLE
Flyp improvements

### PR DESCRIPTION
**Pull request by DarrenWestwood**
* Fix link in altcoin refund email
* Red add refund errors
* Catch all add refund errors
* Update process_alt_response to improve refunds

### CODE REVIEW

**Comment by DarrenWestwood** [comment]
Refunded

File: js/app.js [hunk 1]
```diff
@@ -414,52 +415,67 @@ app.controller('AltcoinController', function($scope, $interval, Order, AltcoinCh
...
+            stop_interval();
+            //Send email if not sent
+            if (send_email) {
+                send_refund_email();
+                send_email = false;
+            }
+        }
+
+        //Refunded
+        if( 'refund_address' in data ){
```
Comment (blockonomics) [inline js/app.js]: multiple if statements can be converted to if elseif wherever possible

**Comment by blockonomics** [comment]
Only send email after user has added refund address:

`Your refund details have been submitted.  The refund will be automatically sent to  {{altrefund}}
If you don\'t get refunded in a few hours, contact <a href="mailto:hello@flyp.me">hello@flyp.me</a> with the following uuid`

We have emailed you a link to this page 
_change to_
We have also emailid you the information on this page

**Comment by DarrenWestwood** [comment]
Refund Submitted New

**Comment by DarrenWestwood** [comment]
Refund Submitted Email

**Pull request merged**